pkg/process/process.go

```diff
@@ -43,6 +43,7 @@ type ProcessInternal struct {
 	// additional internal fields below
 	capabilities *tetragon.Capabilities
 	namespaces   *tetragon.Namespaces
+	nsPid        uint32
```

Comment:
Thanks for the patch, hmm seems we are also calling this nsPid in other places where it should be just Pid!  hmm  will check and get back to you

Reply:
Ok I documented it here https://github.com/cilium/tetragon/pull/775, thanks!